### ISSUE TITLE
Remote invoke of START from BAT kills STARTed process

### ISSUE THREAD

**Issue by DerekBennett** [PowerShell/Win32-OpenSSH#1195]
**"OpenSSH for Windows" version**
This version command did not work for me. The SSH installation I used is the current release: "OpenSSH-Win64-v7.7.0.0p1-Beta"
 
**Server OperatingSystem**
Windows Server 2012 R2 Standard

**Client OperatingSystem**
Windows Server 2012 R2 Standard

**What is failing**
I am using OpenSSH-Win64-v7.7.0.0p1-Beta to invoke a BAT file remotely. The BAT file sets some environment variables and then calls START "" someprogram.exe. This used to work just fine in prior releases of SSH. Now, however, START no longer works. Instead the someprogram.exe instance dies as soon as the BAT file exits.

**Expected output**
I expect the STARTed process to detach and continue running as a parentless process.

**Actual output**
The STARTed process exits as soon as the BAT file that STARTed it exits.

This issue refers to https://github.com/PowerShell/Win32-OpenSSH/issues/1032

**Comment by maertendMSFT** [comment]
By design we associate the job object with the initial process, so that all the child processes are cleaned-up automatically.  This is the same behavior as Linux.
  
Look for a work-around: https://github.com/PowerShell/Win32-OpenSSH/issues/1032

**Comment by DerekBennett** [comment]
@manojampalam asked me on  #1032 to create a new issue, this one. As far as I am aware. The problem I reported is still a problem. The behavior in *nix is that you can execute a remote command that outlives the ssh session. We want the same behavior in Windows. Use a BAT file to set some environment variables and command-line switches, and then run a daemon process that continues without problem after the ssh session goes away.
This issue or #1032 should remain open. Additionally, I do not see a work-around in #1032. If there is one, please write something about the workaround in the documentation.

**Comment by bagajjal** [comment]
Reopening the issue. I will try to reproduce tomorrow.

**Comment by valeriob** [comment]
Hi,
i'm facing the same issue, porting some code from linux to windows, i'm unable to find the same behaviour as nohup :( I tested 8.1 and 8.6.
Anybody found a workaround ?

**Comment by bagajjal** [comment]
@DerekBennett , @valeriob - Have you tried this https://github.com/PowerShell/Win32-OpenSSH/issues/1032#issuecomment-381958274?

By default all the processes launched by SSH are closed once the SSH session is closed. If you want the process to execute independent of ssh session then you need to try spawn your process with special flags as mentioned in the above link.

**Comment by DerekBennett** [comment]
> @DerekBennett , @valeriob - Have you tried this [#1032 (comment)]
No. That solution looks unsustainable. It would be better if SSH would invoke processes with the special flags by default. SSH should have the exact same spawning behavior on all platforms.

**Comment by bagajjal** [comment]
@DerekBennett - It's same behavior on *nix systems as well.
Any process launched within ssh session is tied to that session. In *nix systems, users use screen / tmux to achieve this.

**Comment by valeriob** [comment]
Hi,
i'm currently using cmd, the powershell solution is very obscure and it raises the bar of the requirements (WMI and powershell), if openssh has the same behavior in windows and linux, what would be the nohup equivalent ?

**Comment by bagajjal** [comment]
@valeriob - You need to create process with special flags as mentioned here https://github.com/PowerShell/Win32-OpenSSH/issues/1032#issuecomment-381958274

**Comment by vborioni-onit** [comment]
Hi,
i've done it like suggested, it's not a very nice solution, and it forces to use powershell, i really wish there was a simple tool like nohup.

**Comment by bagajjal** [comment]
@vborioni-onit - Unfortunately windows OS doesn't have nohup, screen, tmux. All these utilities are out of scope of this project.
